### DISCUSSION TITLE
Balancing with Krastorio 2

### DISCUSSION THREAD

**Discussion by Bluebottle39** [Xman1109/Atomic_Overhaul#41] [General]
Hey there,
I was wondering if you have given any thought to balancing with k2?

I am in the middle of a K2SE with atomic overhaul and nuclear fuel (for the breeder reaction).
I understand that this might make things a little more complex to balance but am hoping a discussion might help come up with something.

First off would you consider adding the breeder reaction for the full configuration of the mod? Would be able to remove NF then.

Would you be able to detail your thoughts behind the current balancing of AO?

**Looking at the current ratios, excluding reprocessing and ease of obtaining the elements:**

Uranium cells:
1 U235 is worth 80GJ (same as vanilla)
Calculation:
1xU235 makes 10 cells @ 8GJ.

Plutonium Cell:
1 Pu is worth 5.3GJ
Calculation:
19xPu makes 10 cells @ 10GJ.

MOX Cell:
each Pu is worth 0.66GJ 
each U235 is worth 10.5GJ
Calculation:
45xU235 + 5xPu makes 50 cells @ 12GJ = 480GJ 
base on previous cells 1 PU is worth 1/16 the energy of U235.

This one feels a little underwhelming considering we get so much less energy per element. Is it supposed to be obtained from the waste instead of crafted?

Breeder cells from NF:
1 Pu is worth 40GJ.
Calculation:
1xPu makes 10 cells @ 4GJ.

Anyways the other elements follow the same principals so I will leave them for now.

At this point it would seem like the it would best to use breeder cells and Uranium cells to get the most energy and as a side benefit, lots of extra U235 and PU from the breeding.

In base k2:
Uranium cells:
1 U235 is worth 25GJ.
Calculation:
2xU235 makes 1 cell @ 50GJ.

Now with K2 AO NF:
Uranium cells:
1 U235 is worth 50GJ.
Calculation:
1xU235 makes 1 cell @ 50GJ.
This also makes the modified recipe very odd:
1 xU235 + 19 Graphite makes 10 rods -> 10 rods + 10 empty cells make 1 U Cell

The other change is to breeder cells from NF:
1 PU is worth 50 GJ.
Calculation:
1 Pu makes 2 breeder cells @ 25GJ.

Because the other cells are not adjusted it again seems to make Uranium and Breeder cells the most efficient energy per element.

Let me know if there is anything wrong with my math.
I would love to hear ideas

**Comment by Xman1109** [comment]
Sorry for the late response.
I have given a thought on balancing my mod with other mods, but I just develop my mods as a hobby, and I am starting my apprenticeship in a week.

But I love the suggestions you gave me, that's something I can probably work with.
If my changes take too long for you, you can make a pull request with the changes you suggested. I don't know how long it will take me, but I'll find some time to merge the request and update the mod on the portal.

**Comment by Bluebottle39** [comment]
No worries at all, I appreciate you taking the time to read this.
Oh, that’s exciting, good luck with the apprenticeship!

No rush on any of this.
I'm afraid I don't know much about factorio modding but if I figure it out, I will see what I can do.

A few more ideas:

 - I wonder if it would be a good idea to move MOX fuel after the waste processing. In my mind it would make it clear that it’s there to use up the waste products rather than being crafted.

 - Multiply all fuel values by 6.25 (just like k2 did with the uranium cells) but the recipes makes only 1 cell at a time.
  	- With these changes some other adjustments would be needed:
 		- Cell recipes should only need 1 empty cell and possible change fuel rods to only make and use 1?
   		 - There will be less cells to reprocess so maybe increase research data output from reprocessing?

 - Is there a reason you made Plutonium cells cost so much more? As far as I can tell it can provide similar amounts of energy than uranium so why not make it cost 1 or 2 plutonium per cell? Could lower the amount of Pu you get from reprocessesing Uranium cells to compensate.


As much as I want more U235, the Thorium processing recipe seems very overpowered:
With Uranium Processing recipe you get 560GJ of energy per 1000 uranium ore (7 U235 per 1k).
With Thorium Processing recipe you get between 5280GJ and 13466 GJ per 1000 uranium ore (66 -130 U235 + 0 – 200 Thorium per 1k).
For this I would suggest lowering the chance to get U235 so that the main source can be breeding (as a kovarex replacement).

What do you think?
